### ISSUE TITLE
Missing field "visible" when listening to workspace events

### ISSUE THREAD

**Issue by MTRNord** [JayceFayne/swayipc-rs#20]
The log basically says what the title says:
```
Err(SerdeJson(Error("missing field `visible`", line: 1, column: 1632)))
```

Running the `swayipc-async = "2.0.0-alpha"` version

**Comment by JayceFayne** [comment]
Should be fixed in `2.0.0-alpha.1`

**Comment by MTRNord** [comment]
Thanks! Works :)
